package.json

```diff
@@ -32,7 +32,7 @@
     "@angular/router": "~4.1.3",
     "@angular/upgrade": "~4.1.3",
     "angular2-grid": "2.0.6",
-    "angular2-slickgrid": "git://github.com/Microsoft/angular2-slickgrid.git#1.3.7",
+    "angular2-slickgrid": "git://github.com/Microsoft/angular2-slickgrid.git#1.3.8",
```

Comment:
the most recent vbump for this module broke the restore dialog.  please verify the restore dialog still works with this change.